pages/_app.jsx

```diff
@@ -144,28 +146,24 @@ export default function App(props) {
       <MatomoProvider value={matomoInstance}>
         <StylesProvider jss={jss}>
           <ThemeProvider theme={theme}>
-            <SessionProvider
-              sessionId="pod-browser"
-              restorePreviousSession
-              onSessionRestore={async (url) => {
-                await router.push(url);
-              }}
-            >
-              <FeatureProvider>
-                <AlertProvider>
-                  <ConfirmationDialogProvider>
-                    <CssBaseline />
-                    <div className={bem("app-layout")}>
-                      <PodBrowserHeader />
-
-                      <main className={bem("app-layout__main")}>
-                        <Component {...pageProps} history={history} />
-                      </main>
-                    </div>
-                    <Notification />
-                  </ConfirmationDialogProvider>
-                </AlertProvider>
-              </FeatureProvider>
+            <CssBaseline />
```

Comment:
As far as I know this isn't Session dependent. I'd encourage reordering the providers here based on which updates most frequently as being the deepest provider.